packages/gateway/src/features/registration/type-resolvers.ts

```diff
@@ -549,6 +550,23 @@ export const typeResolvers: GQLResolver = {
       if (!user || !user.valueReference || !user.valueReference.reference) {
         return null
       }
+      const practitionerId = user.valueReference.reference.split('/')[1]
+      const practitioner: fhir.Practitioner = await fetchFHIR(
+        `/Practitioner/${practitionerId}`,
+        authHeader
+      )
+      console.log('WTF: ', JSON.stringify(practitioner))
+      /*  if (
+        !practitioner &&
+        !practitioner.identifier &&
+        !practitioner.identifier[0] &&
+        !practitioner.identifier[0].value
+      ) {
+        throw new Error('Practitioner mobile cannot be found')
+      }
+      // return role from practitioner and the rest from user using mobile to find the user
+      const mobile = practitioner.identifier[0].value*/
```

Comment:
We should get the old role from the practitioner and replace the user's current role with the role that the user had when they were the practitioner on that particular history action